deps/agg/include/agg_pixfmt_rgba.h

```diff
@@ -78,6 +77,64 @@ template<class ColorT, class Order> struct multiplier_rgba
     }
 };
 
+//========================================================multiplier2_rgba
+template<class ColorT, class Order, class Order2 = Order> struct multiplier2_rgba
+{
```

Comment:
I wonder if this addition is needed? Does it offer something that the code @artemp added in https://github.com/mapnik/mapnik/commit/2c931a238f8ff8fefeebfe92890b6e8f9bbb6b05 does not provide?